.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 <!--
 Thank you for reporting an issue.
 
+*IMPORTANT* - please search our issue tracker *first* to ensure the
```

Comment:
`Please search our issue tracker *before* making a new issue. If you cannot find a similar issue, then create a new issue.`